pkg/apis/pipeline/v1alpha1/pipeline_conversion.go

```diff
@@ -25,6 +25,8 @@ import (
 	"knative.dev/pkg/apis"
 )
 
+const FinallyFieldName = "finally"
```

Comment:
I'm a bit confused by why we need this.
Do we ever have a case where we want to convert a beta+ resource back to alpha?
That was the case before we started to store beta, but now it should not be needed anymore.
Else we would end up having to add all checks for all new fields in post alpha?

Reply:
@vdemeester wdyt?

Reply:
This is still the case if you explicitely ask for a v1alpha1 task (that is stored in v1beta1). I don't really why user would do that but tools (older ones) may do that still.